src/dotty/tools/dotc/transform/PatternMatcher.scala

```diff
@@ -50,6 +51,13 @@ class PatternMatcher extends MiniPhaseTransform with DenotTransformer {thisTrans
   override def transformMatch(tree: Match)(implicit ctx: Context, info: TransformerInfo): Tree = {
     val translated = new Translator()(ctx).translator.translateMatch(tree)
 
+    // check exhaustivity and unreachability
+    val engine = new SpaceEngine
```

Comment:
PatternMatcher now has a 🌠SpaceEngine🌠 and can do interstellar travels. Looking forward to battle space invaders.
👾👾👾👾